src/KKManager.Core/Util/ListTools.cs

```diff
@@ -70,16 +70,16 @@ bool SearchStringIsEmpty()
                     PutSearchText();
             };
 
-            var textMatchFilter = new TextMatchFilter(listView);
-            textMatchFilter.StringComparison = StringComparison.OrdinalIgnoreCase;
-            listView.AdditionalFilter = textMatchFilter;
+            var matchFilter = new ModMatchFilter(listView);
+            matchFilter.StringComparison = StringComparison.OrdinalIgnoreCase;
+            listView.ListFilter = matchFilter;
```

Comment:
Using ListFilter instead of AdditionalFilter will cause the filter to be removed when user filters the list by right clicking on columns. It's better to set AdditionalFilter like before and remove ListFilter if you want to show all items.

Reply:
Already use AdditionalFilter.
![preview](https://user-images.githubusercontent.com/75569179/107902204-be0dbb00-6f80-11eb-9413-97d8961ef449.gif)